lib/tasks/rails_db_views_tasks.rake

```diff
@@ -1,7 +1,7 @@
 namespace :db do
 
   desc "Generate all the database views of the current project"
-  task :create_views => :environment do
```

Comment:
This branch is supposed to be compatible with 1.8.7, so I'd prefer to keep the hash rockets.